packages/MarkdownEditor-Core.package/MarkdownHeading.class/instance/fontSizes.st

```diff
@@ -0,0 +1,4 @@
+accessing
+fontSizes
+	
+	^ Dictionary newFrom: {1->34. 2->28. 3->22. 4->19. 5->16. 6->14}
```

Comment:
Wouldn't be an `OrderedCollection` sufficient?

Reply:
I thought this too, but I thought it is nice to see that it is a mapping from the level to the size. Hinding the index by using the `OrderedCollection` would hind this mapping because it hides the index. Thus I like it better with the `Dictionary`.

Reply:
Did also thought of semantics, but looking at the line, i only see numbers and it is hard to difference the keys from the values... I find the second option with spaces around the arrows easier to read, but as this is subject-dependant, I'd rather open a poll than discuss long.

```smalltalk
Dictionary newFrom: {1->34. 2->28. 3->22. 4->19. 5->16. 6->14}
Dictionary newFrom: {1 -> 34. 2 -> 28. 3 -> 22. 4 -> 19. 5 -> 16. 6 -> 14}
{34. 28. 22. 19. 16. 14}
```

(okay, that's not really an `OrderedCollection` but an `Array`)